ajax/libs/blendui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "blendui",
+  "filename": "BlendHybridUI.min.js",
+  "author": "Clouda+ team",
+  "description": "A Hybrid Javascript Framework For Mobile WebApp",
+  "homepage": "http://clouda.baidu.com/blendui/introduction/introduction",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Clouda-team/BlendUI.git"
+  },
+  "version": "0.0.4",
+  "keywords": [
+    "Native",
+    "Javascript",
+    "webview",
+    "web"
+  ],
+  "npmName": "blendui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(BlendHybridUI-[0-9].[0-9].[0-9]*)"
```

Comment:
Are you sure about there will only be a single number? Why do you use`+([0-9])` instead of `[0-9]`?